pkg/provisioner/fixture/fixture.go

```diff
@@ -17,7 +17,7 @@ var provisionRequeueDelay = time.Second * 10
 
 // Provisioner implements the provisioning.Provisioner interface
 // and uses Ironic to manage the host.
-type fixtureProvisioner struct {
+type Provisioner struct {
```

Comment:
What's the point of renaming this?

Reply:
To allow setting the ready flag from the test

Reply:
The type is private and only accessible via an interface that doesn't have the SetIsReady() function, so renaming it makes it public but still requires a type coercion in the test. What if we move it up into pkg/provisioner instead? That would let us keep it private, and the tests could instantiate one directly to avoid the type coercion.

Reply:
If it is not a problem to move it then I think it sounds fine

Reply:
Renamed the struct, see comment below